components/analytics/usage-history.tsx

```diff
@@ -67,22 +65,16 @@ interface IUsageHistory {
 }
 
 export function UsageHistory() {
-  const [usageHistory, setUsageHistory] = useState<IUsageHistory[] | null>(null);
+  const [usageHistory, setUsageHistory] = useState<IUsageHistory[]>([]);
   const { showToast } = useToast();
-  const { settings } = useSettings();
   useEffect(() => {
-    if (!settings.loggedIn) {
-      setUsageHistory([]);
-      return;
-    }
-    setUsageHistory(null);
     (async () => {
       let stacksPromise = await getAllStacks().catch(ex => {
-        showToast("Error occurred", "error");
+        showToast("Error Occured", "error");
```

Comment:
it's spelled `occurred` if you want to just capitalized the first letter of it to `Occurred` I don't mind.